httpserver/common.h

```diff
@@ -37,9 +67,9 @@ template <>
 struct hash<sn_record_t> {
     std::size_t operator()(const sn_record_t& k) const {
 #ifdef INTEGRATION_TEST
-        return hash<uint16_t>{}(k.port);
+        return hash<uint16_t>{}(k.port());
 #else
-        return hash<std::string>{}(k.address);
+        return hash<std::string>{}(k.address());
```

Comment:
This (and the others further down) should actually be `sn_address()` but I think we should go back to having the getter as just `address()`, we already know it is an `sn_record_t`

Reply:
My issue with `address` is that it might be perceived as "ip address" or something of that nature. I think `sn_address` (or loki address?) should make it a bit more clear.

Reply:
Yeah that's fair, just renaming these to `sn_address` is fine then

Reply:
Updated.